frontend/src/lib/components/LLMButton/ExplainCSPViolationButton.tsx

```diff
@@ -14,13 +16,16 @@ export const ExplainCSPViolationButton = ({
     ...buttonProps
 }: ExplainCSPViolationButtonProps): JSX.Element => {
     const [loading, setLoading] = useState(false)
+    const [isOpen, setIsOpen] = useState(false)
+    const [description, setDescription] = useState('')
 
     const handleClick = async (): Promise<void> => {
         setLoading(true)
         try {
             const r = await api.cspReporting.explain(properties)
             if (r) {
-                lemonToast.success(r.response)
+                setDescription(r.response)
```

Comment:
<img width="183" alt="Screenshot 2025-05-17 at 11 22 03" src="https://github.com/user-attachments/assets/ee719e06-3371-4d14-8d7c-54bee35db64c" />

I guess we can move the loading state entirely into the popover now

so `<spinner> thinking...`

nicer than the button changing shape